dessia_common/core.py

```diff
@@ -125,7 +125,17 @@ def __init__(self, name: str = '', **kwargs):
     def base_dict(self):
         """ Base dict of the object, with just its name. """
         dict_ = SerializableObject.base_dict(self)
-        dict_['name'] = self.name
+        return dict_
+
+    def _serializable_dict(self):
```

Comment:
Why not injecting name in Serializable object instances ? Currently, classes who are going to inherit from Serializable won't benefit from this change.

Also, it might have implications on frontend, as I don't know what happened if name is undefined (and not defined with empty string)

Reply:
name is not an attribute of serializable object

Reply:
That is why I commented. Should we add it to SerializableObject ?